hypertools/tools/procrustes.py

```diff
@@ -66,15 +65,8 @@ def fit(source, target):
 
         for i, ds in enumerate((source, target)):
             data = ds
-            if _demean:
-                if i == 0:
-                    mean = _offset_in
-                else:
-                    mean = data.mean(axis=0)
-                data = data - mean
```

Comment:
```python
mean = _offset_in if i == 0 else data.mean(axis=0)
data -= mean
```

Reply:
_demean is always False so i removed the whole statement

Reply:
oop, i did it wrong though, hang on